photon-ml/src/main/scala/com/linkedin/photon/ml/cli/game/training/Driver.scala

```diff
@@ -514,18 +538,22 @@ final class Driver(val params: Params, val sparkContext: SparkContext, val logge
 
     trainingDataSet.foreach { case (_, rddLike: RDDLike) => rddLike.unpersistRDD() }
 
+    timer.start()
+    val bestModel = selectBestModel(gameModelsMap, validatingDataAndEvaluatorsOption)
+    timer.stop()
+    logger.info(s"Time spent in model evaluation: ${timer.durationSeconds} (s)\n")
```

Comment:
I am not sure that this should go inside the block - regardless of whether the users want to see an output model, selecting the best model is now part of the training/validation algorithm, IMHO.